scripts/get-channels.ts

```diff
@@ -0,0 +1,23 @@
+import versions from '../configs/versions.json';
+import minimist, {ParsedArgs} from 'minimist';
+
+interface Args extends ParsedArgs {
+  amp_version?: string;
+}
+const {amp_version: ampVersion}: Args = minimist(process.argv.slice(2));
+
+function getChannels(): string[] {
+  if (!ampVersion) {
+    return Object.keys(versions);
+  }
+
+  const channels = [];
+  for (const [channel, version] of Object.entries(versions)) {
+    if (version && version.slice(-13) == ampVersion) {
+      channels.push(channel);
+    }
+  }
+  return channels;
+}
+
+console.log(getChannels());
```

Comment:
Supernit feel free to ignore: I feel like a CLI script whose output is explicitly passed via pipe should use `process.stdout.write` instead of `console.log`, cause theoretically¹ `console.log` could be augmented to add things like log time, etc.

¹ love me a programming theory